.github/workflows/build.yml

```diff
@@ -2,17 +2,27 @@ name: Descent 3 Build
 
 on:
   push:
-    branches: [ "main" ]
   pull_request:
     branches: [ "main" ]
 
 jobs:
   build:
+
+    # In forks - all pushes and pull requests
+    # In main repo - only pushes and pull requests to the main branch
+    if: ( github.repository != 'kevinbentley/Descent3' )
+     || ( github.repository == 'kevinbentley/Descent3'
+          && ( github.ref == 'refs/heads/main'
+            || github.event_name == 'pull_request' ) )
+
```

Comment:
It's better to rework on top to
```
on:
  push:
    branches:
      - main
  pull_request:
    branches:
       - main
```
Don't limit CI on own forks.

Reply:
I don't understand, this change is actually needed to run CI on own forks in all branches, not limited to main. `on` mechanics doesn't allow to switch logic. With `on` we only can make CI to run on all branches in origin repository too, though. My intention was just to make it run CI in linux-build branch on my fork, and don't change behavior in upstream. It is not needed actually, anymore, so I can remove this change from PR.

Reply:
Yes, please remove it.